@types/interfaces.d.ts

```diff
@@ -0,0 +1 @@
+export type ScrollerScrollDirection = 'horizontal' | 'vertical' | 'none' | null | undefined;
```

Comment:
I added `null` and `undefined` here instead of in the JSDoc, because they are explicitly mentioned in the docs as a valid scroll direction.